daisy_workflows/build-publish/rhui/install_cds.sh

```diff
@@ -89,6 +89,15 @@ cat $tempdir/rhui-tools.conf > /etc/rhui/rhui-tools.conf
 # Remove enrollment cert and repos from final image.
 subscription-manager remove --all
 
+# Install health checks.
+install -D -t /opt/google-rhui-infra $tempdir/health_check.py
+for unit in rhui-health-check.{service,timer}; do
+  # RH recommends 664 for unit files
+  # https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/configuring_basic_system_settings/assembly_working-with-systemd-unit-files_configuring-basic-system-settings
```

Comment:
it's a nit but that's neither backed up by the link nor is RH in any sense a reputable source of truth for things like this. i don't care what permissions we use (it literally just needs to be readable by root) but let's strike the comment

Reply:
makes sense -- removed the RH link